db/ignore_patterns/internetcentrum.json

```diff
@@ -43,8 +49,10 @@
     "modules\\.php\\?name=coppermine.*meta=(?:topn|toprated|lastcom|lastup|lastupby|random|lastcomby)",
     "modules\\.php\\?name=Statistics",
     "name=Kalender",
+    "name=Statistics",
```

Comment:
Are there URLs that have just `name=Statistics`, aren't `modules.php?name=Statistics`, but produce similar content?

(I've not seen them but that definitely doesn't mean anything)